crates/toml_edit/src/de/value.rs

```diff
@@ -28,6 +28,9 @@ use crate::de::Error;
 /// assert_eq!(config.title, "TOML Example");
 /// assert_eq!(config.owner.name, "Lisa");
 /// ```
+///
+/// Already parsed [`crate::Value`] types can be turned into [`ValueDeserializer`]
```

Comment:
Does `crate::Value` get rendered literally in the rustdoc?  That  might not read all that well and we should probably take the `crate::` out of the link text